src/lwc/geBatchGiftEntryHeader/geBatchGiftEntryHeader.html

```diff
@@ -10,10 +10,11 @@
                                   data-action={ACTIONS.DRY_RUN_BATCH}
                                   data-qa-locator={qaLocatorBatchDryRun}>
                 </lightning-button>
-                <lightning-button label={LABELS.bgeProcessBatch}
+                <lightning-button label={processBatchButtonName}
                                   onclick={handleClick}
                                   data-action={ACTIONS.PROCESS_BATCH}
-                                  data-qa-locator={qaLocatorProcessBatch}>
+                                  data-qa-locator={qaLocatorProcessBatch}
+                                  aria-label={processBatchButtonAltName}>
```

Comment:
Hey @bdvorachek - Hoping you might be the best person to connect with on this. My work item (https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B00000094E17IAE/view) calls to add "alt-text" to the "Process Batch" button here for the purposes of supporting screen readers.

In my research, it looks like the LWC "lightning-button" component doesn't support this, so I'm using "aria-label" instead. Do you know if this is the ideal workaround or would you happen to have another recommendation on the best way to move forward here? Thanks!

Reply:
I think `aria-description` or `aria-describedby` is more appropriate if we're providing a longer-form description of what the button does. I'd have to look at the behavior in a screen reader, but I think `aria-label` ends up clashing with the regular `label` attribute.